dll/win32/shell32/CQueryAssociations.cpp

```diff
@@ -96,6 +97,11 @@ HRESULT STDMETHODCALLTYPE CQueryAssociations::Init(
     {
         WCHAR *progId;
         HRESULT hr;
+        LPCWSTR pchDotExt;
+
+        pchDotExt = PathFindExtensionW(pszAssoc);
```

Comment:
Can you show us pszAssoc string examples for which this function call is actually needed?
(This would help the reviewers of this PR to also understand with which kind of strings CQueryAssociations::Init function can be called.)

Reply:
For example: `C:\Program Files\My Program\ReadMe.txt`